test_create_release_notes.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import os
+import yaml
+from create_release_notes import ReleaseNoteGenerator
+import tempfile
+import filecmp
+import shutil
+
+
+class TestReleaseNotesGenereator(unittest.TestCase):
+    """
+    Test cases for ReleaseNoteGenerator. Run using
+        python3 -m unittest test_create_release_notes.py
+    Tests: 
+    1. Release notes are empty file
+    2. Release notes are non-empty file
+    3. Changelog has different headline format
+    """
+
+    def load_data(self, input):
+        self.maxDiff = None
+
+        with open("test/" + input + "_input.yaml") as input_file:
+            self.input_data = yaml.load(input_file, Loader=yaml.FullLoader)
+        for key, value in self.input_data.items():
+            os.environ[key] = value
+        self.tmpfile = tempfile.NamedTemporaryFile()
+        os.environ['GITHUB_OUTPUT'] = self.tmpfile.name
+
+        with open("test/" + input + "_validate.yaml") as validate_file:
+            self.validate_data = yaml.load(
+                validate_file, Loader=yaml.FullLoader)
+        shutil.copyfile(f"test/{input}_release_notes.md",
+                        f"test/{input}_release_notes_copy.md")
+        self.validate_data['RELEASE_NOTES_PATH'] = f"test/{input}_release_notes_copy.md"
+        self.validate_data['RELEASE_NOTES_VALIDATE_PATH'] = f"test/{input}_release_notes_validate.md"
```

Comment:
Proc to tady ukladas do toho dictionary `validate_data` a nemas to jako dalsi promennou? Nemusel bys tak vsude kopirovat ten index name